src/rdb/rdb_raft.c

```diff
@@ -1119,54 +1119,139 @@ rdb_raft_update_node(struct rdb *db, uint64_t index, raft_entry_t *entry, rdb_vo
 /* See rdb_raft_log_offer_single. */
 #define RDB_RAFT_ENTRY_NVOPS 2
 
+/* If entry == NULL, the caller doesn't want to apply the entry. */
+static int
+rdb_raft_entry_count_vops(struct rdb *db, raft_entry_t *entry)
+{
+	int count = 0;
+
+	/* Count those that will be invoked when applying the entry. */
+	if (entry->type == RAFT_LOGTYPE_NORMAL) {
+		int rc;
+
+		rc = rdb_tx_count_vops(db, entry->data.buf, entry->data.len);
+		if (rc < 0)
+			return rc;
+		count += rc;
+	} else if (raft_entry_is_cfg_change(entry)) {
+		count += RDB_RAFT_UPDATE_NODE_NVOPS;
+	} else {
+		D_ERROR(DF_DB ": unknown entry type %d\n", DP_DB(db), entry->type);
+		return -DER_IO;
+	}
+
+	/* Count those that will be invoked when storing the entry. */
+	count += RDB_RAFT_ENTRY_NVOPS;
+
+	return count;
+}
+
```

Comment:
[Note] This function is moved (without changes, I think) upward from old line 1219, because we now call it in `rdb_raft_log_offer_single` instead of `rdb_raft_cb_log_offer`.